ethereum/blockwatch/stack_test.go

```diff
@@ -0,0 +1,99 @@
+package blockwatch
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/0xProject/0x-mesh/common/types"
+	"github.com/0xProject/0x-mesh/db"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	miniHeaderOne = &types.MiniHeader{
+		Number:    big.NewInt(1),
+		Hash:      common.HexToHash("0x1"),
+		Parent:    common.HexToHash("0x0"),
+		Timestamp: time.Now().UTC(),
+	}
+	miniHeaderTwo = &types.MiniHeader{
```

Comment:
We don't appear to use this. Let's delete it unless we need the extra miniheader for a test.